PersonBook/PersonBook/PersonBook/Service/IPersonService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace PersonBook
+{
+    interface IPersonService<Person>
+    {
+        public List<Person> ListPersons();
```

Comment:
You can use ICollection instead of List!

Reply:
You shouldn't use `public` in interfaces!!!

Reply:
Done